src/Components/Common/Menu.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { Navbar, Nav, Button, NavDropdown, Form, FormControl } from 'react-bootstrap';
+import { Link } from 'react-router-dom';
+const Menu = () => {
+  return (
+    <Navbar bg="light" expand="lg">
+    <Navbar.Brand href="#home">React-Bootstrap</Navbar.Brand>
```

Comment:
Why does this say React-Bootstrap. Please use our project name for this. And fix the href to point to the a real page instead #home.